microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md

```diff
@@ -0,0 +1,136 @@
+
+Customizing and simplifying the developer experience includes making our tools available to everyone, regardless of their preferred language.  We've added two new languages, Czech and Vietnamese, to our list of supported languages, and will keep adding more.  If your preferred language isn't supported in DevTools, let us know!
+
+![DevTools in Czech and Vietnamese.](devtools-100-images/czech-vietnamese.png)
+
+For more information, see [Change DevTools language settings](../../../customize/localization.md).
+
+
+<!-- ====================================================================== -->
+## The Network tool now displays how a request was fulfilled
+
+<!-- Title: You no longer have to wonder if a request was fulfilled by your service worker or cache -->
+<!-- Subtitle: The "Fulfilled by" column in the Network tool tells you how a request was fulfilled. -->
+
+In Microsoft Edge 100, the **Network** tool supports a new, **Fulfilled by** column in the request log.  The **Fulfilled by** column shows how a request was fulfilled.  If the request was fulfilled by your service worker or from the cache, that information is now tracked in the **Fulfilled by** column.  When exporting or importing the request log as a HAR file, the information in the **Fulfilled by** column is also included.
+
+![The Fulfilled By column in the request log of the Network tool.](devtools-100-images/fulfilled-by-request-log.png)
+<!--
+If you don't have the **Fulfilled by** column, right-click the table headers in the request log and make sure **Fulfilled by** is checked.
+-->
+
+To learn more about the request log in the Network tool, see [Filter requests by properties](../../../network/reference.md#display-a-log-of-requests) in _Network features reference_.  For more information about service workers, see [Service Worker improvements](../../../service-workers/index.md).
+
+For the history of this feature, see [Outlook: Add column in the Network activity log to track why resources are not loaded from DiskCache when ServiceWorker is present · Issue #16](https://github.com/MicrosoftEdge/DevTools/issues/16).
+
+
+<!-- ====================================================================== -->
+## When importing a performance profile, links now map to your original code
+
+<!-- Title: Use sourcemaps from Azure Artifacts symbol server to better debug performance issues -->
+<!-- Subtitle: Links from an imported performance profile now map to your original code because of source maps. -->
+
+When recording a performance profile, the flame chart in the **Performance** tool displays links from each event on the main thread to the corresponding JavaScript function in the **Sources** tool.  However, these links are broken when you export and import a performance profile.
+
+In Microsoft Edge 100, the links from an imported performance profile now use source maps from Azure Artifacts symbol server to map back to your familiar, original source code.
```

Comment:
Suggest adding:

> The performance profile must have been exported in Microsoft Edge 100, which includes the information necessary to resolve source maps in the exported performance trace.

This is so people can understand that exporting from < 100 and importing into 100 won't allow it to magically work.